.prettierrc.json

```diff
@@ -0,0 +1,6 @@
+{
+    "semi": true,
+    "trailingComma": "none",
```

Comment:
just out of curiosity, why did we decide against trailing commas?

Reply:
Not sure. Personally, I'm all for trailing commas. It makes adding properties much easier.

Reply:
I'm in favor of trailing commas as well